tests/test_file_connector.py

```diff
@@ -0,0 +1,128 @@
+"""Python Module that contains TestFileConnector class"""
+from unittest.mock import patch, mock_open
+
+import pytest
+
+from happy_bank_core.data.file_connector import FileConnector
+from happy_bank_core.logic.account import Account
+
+
+READ_TEST_DATA = (
+    '{"101": {"id": "101", "name": "John Doe", "deposit": 1000}, '
+    '"102": {"id": "102", "name": "Johanna Doe", "deposit": 1000}}'
+)
+
+
+class TestFileConnector:
+    """Class that contains FileConnector unittests"""
+
```

Comment:
Add setup_class method where you create a accounts dictionary filled with READ_TEST_DATA to avoid code duplication and in place values.